### PR TITLE
docs: improve `multiset_delta()` docs

### PR DESCRIPTION
**Pull request by MingweiSamuel**

### CODE REVIEW

File: hydroflow_lang/src/graph/ops/multiset_delta.rs [hunk 1]
```diff
@@ -5,30 +5,43 @@ use super::{
...
+/// The multiset inverse of [`persist()`](#persist).
+///
+/// > 1 input stream of `T`, 1 output stream of `T`, where `T: Eq + Hash`
+///
+/// For set semantics, [`unique()`](#unique) can be thought of as a "delta" operator, the inverse
+/// of [`persist()`](#persist). In `persist`, new items come in, and all items are repeatedly
+/// released out. Conversely, `unique` take repeated items in, and only releases the new ones out.
+///
+/// This operator does a similar inversion but for multiset semantics, with some caveats. When it
+/// receives duplicate items, instead of ignoring them, it "subtracts" them from the items received
```
Comment (jhellerstein) [inline hydroflow_lang/src/graph/ops/multiset_delta.rs:17]: "subtracts" is confusing. Why not say "if we received `k` copies of an item in the previous tick, and we receive `l > k` copies in the current tick, we output `l - k` copies of the item."

File: hydroflow_lang/src/graph/ops/multiset_delta.rs [hunk 2]
```diff
@@ -5,30 +5,43 @@ use super::{
...
+/// in the previous tick, and only releases new items if there are more than in the previous tick.
+/// However unlike `unique`, this count is only maintained for the previous tick, not over all time.
```
Comment (jhellerstein) [inline hydroflow_lang/src/graph/ops/multiset_delta.rs:19]: Given the proposed rewording above, maybe rephrase to match.

I'm actually confused as to whether it remembers the MAX number of copies over time, or just the last tick number?  If it's MAX, then we should probably implement a generic `lattice_delta`, where this is a specific one for multiset lattices. If it's not MAX, then I'd like to discuss semantics --- it'd be weird for this to be the only 1-tick windowed operator (other than `next_tick`).

File: hydroflow_lang/src/graph/ops/multiset_delta.rs [hunk 3]
```diff
@@ -5,30 +5,43 @@ use super::{
...
-/// input_send.send(3).unwrap();
+/// input_send.send('a').unwrap();
+/// input_send.send('c').unwrap();
+/// input_send.send('a').unwrap();
+/// input_send.send('a').unwrap();
 /// flow.run_tick();
-/// // 5, 3
-/// // First two "3"s are removed due to previous tick.
+/// // 'c', 'a'
+/// // First two 'a's are removed due to previous tick.
```
Comment (jhellerstein) [inline hydroflow_lang/src/graph/ops/multiset_delta.rs:44]: Give an example of the 3rd tick to clarify what confuses me above about MAX over all time vs last tick?

**Review by jhellerstein — COMMENTED** [review]
Please revise, as I'm still finding it ambigous

File: hydroflow_lang/src/graph/ops/multiset_delta.rs (hunk 2 above)
Comment (MingweiSamuel) [reply hydroflow_lang/src/graph/ops/multiset_delta.rs:19]: It is only the previous tick. The semantics become weird if it's not somewhere after a `persist` operator
